tests/e03_emodeBorrow.spec.ts

```diff
@@ -18,19 +20,24 @@ import {
   assertBankrunTxFailed,
   assertI80F48Approx,
 } from "./utils/genericTests";
-import { CONF_INTERVAL_MULTIPLE } from "./utils/types";
+import {
+  CONF_INTERVAL_MULTIPLE,
+} from "./utils/types";
 import { getBankrunBlockhash } from "./utils/spl-staking-utils";
 import { deriveBankWithSeed } from "./utils/pdas";
-import { wrappedI80F48toBigNumber } from "@mrgnlabs/mrgn-common";
+import {
+  wrappedI80F48toBigNumber,
+} from "@mrgnlabs/mrgn-common";
 import { USER_ACCOUNT_E } from "./utils/mocks";
-import { accountInit, borrowIx, depositIx } from "./utils/user-instructions";
+import { accountInit, borrowIx, composeRemainingAccounts, depositIx } from "./utils/user-instructions";
 
+// Banks are listed here in the sorted-by-public-keys order - the same used in the lending account balances
```

Comment:
I don't think this is possible, the emode group keypair is random per test execution so there's no predictable order to the derived keys where the group is a seed.

Reply:
Nevermind I see you made the seed non-random, that's probably a good idea lol.